automation/src/test/scala/org/broadinstitute/dsde/firecloud/test/library/DataLibrarySpec.scala

```diff
@@ -51,4 +51,43 @@ class DataLibrarySpec extends FreeSpec with WebBrowserSpec with UserFixtures wit
       }
     }
   }
+
+
+  /**
+    * Test creates two workspaces. Only one workspace has tags
+    */
+  "Input multiple tags in filter field" in withWebDriver { implicit driver =>
+    val curatorUser = UserPool.chooseCurator
+    implicit val authToken: AuthToken = curatorUser.makeAuthToken()
+    withWorkspace(namespace, "DataLibrarySpec", attributes = Some(WorkspaceData.tagAC)) { wsName =>
```

Comment:
You could use the workspace name as part of the tag, in order to ensure it's unique, and pass that in here.

Reply:
good idea. I'll make the change.

Reply:
don't know how to do this in Scala. `wsName` is generated inside `withWorkspace` function that need parameter named `attributes`